Source/Core/Runtime/Behaviors/ScalingBehavior.cs

```
@@ -73,26 +73,28 @@ public override void Start()
             /// <inheritdoc />
             public override IEnumerator Update()
             {
-                float startedAt = Time.time;
+                if (Data.Targets.Values.Count() == 0)
```

You could use Data.Targets.HasValue() here instead. Trying to count Values will return a NRE if Values itself is null.

Also I'm not sure we want to throw an exception if there is a valid reference and zero values. Referencing an object group which for some reason has zero objects in the scene might be a legitimate use case. Maybe you could just check if Values == null?